elements/notification-container/notification-container.js

```diff
@@ -0,0 +1,86 @@
+import BulbsElement from 'bulbs-elements/bulbs-element';
+import NotificationDisplay from './notification-display';
+import invariant from 'invariant';
+import React, { PropTypes } from 'react';
+import { registerReactElement } from 'bulbs-elements/register';
+
+
+var localStorageMixin = require('react-localstorage');
+var reactMixin = require('react-mixin');
+
+
+class NotificationContainer extends BulbsElement {
+
+  constructor (props) {
+    invariant(!!props.src, 'notification-container component requires a src');
+    super(props);
+    this.state = {
+      next: null,
+      local_notification_ids: [],
+      notification: {}
+    };
+  }
+
+  getStateFilterKeys () {
+    return ['local_notification_ids'];
+  }
+
+  requestNotifications (src) {
+    fetch(src)
+      .then(function(response) {
+        return response.json();
+      })
+      .then(this.handleRequestSuccess.bind(this));
+  }
+
+  componentDidMount () {
+    this.requestNotifications(this.props.src);
+  }
+
+  handleRequestSuccess (response) {
+    console.log(response);
+    var notifications = response.results;
+    this.setState({ next: response.next });
+
+    if (this.state.local_notification_ids.length > 0) {
+      notifications = this.removeLocalNotifications(notifications);
+    }
+
+    if (notifications.length > 0) {
+      this.setState({ notification: notifications[0] });
+      this.setState({
+        local_notification_ids: this.state.local_notification_ids.concat([notifications[0].id])
+      });;
+    } else if (this.state.next !== null) {
+      this.requestNotifications(this.state.next);
+    }
```

Comment:
It doesn't look like next is used outside this function.

I think you could avoid setting it as state and 

```js
else if (response.next !== null) {
  this.requestNotifications(response.next);
}
```